src/main/java/edu/wpi/first/pathui/MainController.java

```diff
@@ -11,7 +11,7 @@
 import javafx.scene.input.KeyEvent;
 import javafx.scene.layout.Pane;
 
-
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
Okay we need a solution to this. The other controller is over by 1 function as well. You need to find a reasonable way to subcategorize some of the functions in MainController and move them into a helper class. It is pretty reasonable to imagine we are going to want to add more functionality to the MainController. This will get out of hand if we let it.

I will put the same note on PathDisplayController.